packages/ui-kit/src/design-system.js

```diff
@@ -126,6 +126,7 @@ export const dimensions = {
   heights: {
     header: '48px',
     inputSearch: '32px',
+    inputSearchSmall: '26px',
```

Comment:
Why do we have two heights here?

Reply:
The first one is for the searchbox in the childsection navigator. Maybe I could rename it to make it clearer